files_airflow_ext/dot-gitignore

```diff
@@ -5,7 +5,9 @@
 
 *
 
-# Allow this gitignore, airflow.cfg, webserver_config.py to be checked in to the repo.
+# Allow some explicit files and paths to be checked in
```

Comment:
Originally we didn't know much about the files needing to be stored here. If you want to switch the paradigm and specify just the files-to-be-ignored, that's also totally fine, I think.